Description: Skip 3 tests that are known to fail
 because we are using a newer system ICU than the ICU bundled
 with mozjs38 or tested with mozjs38, confirmed by
 Philip Chimento
Author: Jeremy Bicha <jbicha@ubuntu.com>
---
 js/src/tests/Intl/DateTimeFormat/format.js                             |    2 +-
 js/src/tests/ecma_6/String/normalize-generateddata-part1-not-listed.js |    2 +-
 js/src/tests/test262/intl402/ch10/10.2/10.2.3_b.js                     |    1 +
 3 files changed, 3 insertions(+), 2 deletions(-)

--- a/js/src/tests/Intl/DateTimeFormat/format.js
+++ b/js/src/tests/Intl/DateTimeFormat/format.js
@@ -1,4 +1,4 @@
-// |reftest| skip-if(!this.hasOwnProperty("Intl"))
+// |reftest| fails
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
--- a/js/src/tests/ecma_6/String/normalize-generateddata-part1-not-listed.js
+++ b/js/src/tests/ecma_6/String/normalize-generateddata-part1-not-listed.js
@@ -1,4 +1,4 @@
-// |reftest| skip-if(!xulRuntime.shell) -- uses shell load() function
+// |reftest| fails
 
 var BUGNUMBER = 918987;
 var summary = 'String.prototype.normalize - not listed in part1';
--- a/js/src/tests/test262/intl402/ch10/10.2/10.2.3_b.js
+++ b/js/src/tests/test262/intl402/ch10/10.2/10.2.3_b.js
@@ -1,3 +1,4 @@
+// |reftest| fails
 // Copyright 2012 Mozilla Corporation. All rights reserved.
 // This code is governed by the license found in the LICENSE file.
 
